scorers/classification/binary/kolmogorov_smirnov.py

```diff
@@ -0,0 +1,97 @@
+"""Kolmogorov-Smirnov scorer recipe.
+
+If you need to print debug messages into DAI log, uncomment lines with logger and loggerinfo.
+
+Starting 1.10.2 - DAI handles exceptions raised by custom scorers.
+Default DAI behavior is to continue experiment in case of Scorer failure.
+To enable forcing experiment to fail, in case of scorer error, set following parameters in DAI:
+  - skip_scorer_failures=false (Disabled)
+  - skip_model_failures=false (Disabled)
+"""
+
+import numpy as np
+from scipy.stats import ks_2samp
+import typing
+from h2oaicore.metrics import CustomScorer
+from h2oaicore.systemutils import print_debug
+import logging
+from h2oaicore.systemutils import config
+from h2oaicore.systemutils import make_experiment_logger, loggerinfo, loggerwarning
+
+
+class KolmogorovSmirnov(CustomScorer):
+    _description = NotImplemented
+    _maximize = True  # whether a higher score is better
+    _perfect_score = 1.0  # the ideal score, used for early stopping once validation score achieves this value
+    _supports_sample_weight = False  # whether the scorer accepts and uses the sample_weight input
+    _binary = True
+    _regression = False
+    _multiclass = False
+    _display_name = "Kolmogorov_Smirnov"
+
+    def __init__(self):
+        CustomScorer.__init__(self)
+
+    @staticmethod
+    def do_acceptance_test():
+        """
+        Whether to enable acceptance tests during upload of recipe and during start of Driverless AI.
+        Acceptance tests perform a number of sanity checks on small data, and attempt to provide helpful instructions
+        for how to fix any potential issues. Disable if your recipe requires specific data or won't work on random data.
+        """
+        return False
```

Comment:
why disabled? isn't it generic enough to work?

Reply:
It is generic scorer, should work for all cases. Will make a change and push